spikeinterface/core/waveform_tools.py

```diff
@@ -47,7 +47,7 @@ def allocate_waveforms(recording, spikes, unit_ids, nbefore, nafter, mode='memma
         shape = (n_spikes, nsamples, num_chans)
         
         if mode =='memmap':
-            filename = folder / f'waveforms_{unit_id}.npy'
+            filename = str(folder / f'waveforms_{unit_id}.npy')
```

Comment:
it needs to be a raw str!

Reply:
what ??

Reply:
`filename = rf"{str(folder / f'waveforms_{unit_id}.npy')}"`

Reply:
No, I don't think this is the problem.

Reply:
Have you given it a try?